source/features/recently-pushed-branches-enhancements.tsx

```diff
@@ -47,7 +47,7 @@ async function init(): Promise<false | void> {
 	document.body.classList.add('rgh-recently-pushed-branches');
 
 	// Move or add list next to the notifications bell
-	select('.Header-item--full,.HeaderMenu nav')!.after(widget);
+	select.last('.Header-item--full, .HeaderMenu nav')!.after(widget);
```

Comment:
The reason it was off the screen before is in smaller views is uses the second one.